packages/uops-dashboard/next.config.mjs

```diff
@@ -3,4 +3,5 @@ const nextConfig = {
   reactStrictMode: true,
 }
 
+// biome-ignore lint/style/noDefaultExport: <explanation>
```

Comment:
<img width="435" alt="image" src="https://github.com/user-attachments/assets/4ecc2c4a-7f9f-4c6a-bc45-f3c1a28ef492">

Reply:
Fixed, was not referencing root biome config file